cfgov/v1/auth_forms.py

```diff
@@ -36,14 +36,6 @@ class UserEditValidationMixin(PasswordValidationMixin):
     user_attribute = 'instance'
 
 
-class CFGOVUserCreationForm(UserEditValidationMixin, UserCreationForm):
-    pass
```

Comment:
We probably shouldn't be breaking those things though. I'm all for using Django as much as we can, but if there's a requirement (especially dealing with users resetting passwords which seems to be a frequent occurrence) then it should remain unbroken.

_waits for counter arguments_